api/History.json

```diff
@@ -0,0 +1,393 @@
+{
+  "api": {
+    "History": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/History",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": true
+          },
+          "firefox_android": {
+            "version_added": true
+          },
+          "ie": {
+            "version_added": true
+          },
+          "opera": {
+            "version_added": true
+          },
+          "opera_android": {
+            "version_added": true
+          },
+          "safari": {
+            "version_added": true
+          },
+          "safari_ios": {
+            "version_added": true
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "length": {
```

Comment:
All the subfeatures are missing status info, this should probably be fixed :)

Reply:
Good catch once more, @connorshea. I have added these now (fa90807).